feaT(demo):WIP config simplifaction via. luster API

stephenatwell opened the pull request:
merging all our recent setup updates

parth-srivastav committed:
Adding Argo Rollouts to the demo script

parth-srivastav committed:
Added Installation for the kubectl plugin

stephenatwell committed:
Create deploy-w-argo.yml

stephenatwell committed:
Create argoonly.yml

stephenatwell committed:
Create argo-facts.yml

stephenatwell committed:
Create argo-facts-3.yml

stephenatwell committed:
Update deploy3.yml

parth-srivastav committed:
Create Rollouts dashboard in the namespace

parth-srivastav committed:
make dashboard run in background

Commit:
updated argo deploy logic

Commit:
Merge branch 'argo-install' of https://github.com/armory-io/borealis-demo into argo-install

parth-srivastav committed:
Adding cmdhook manifest and deploy command during setup

parth-srivastav committed:
Move Argo Installation execution after RNA install

parth-srivastav committed:
Merge branch 'argo-install' of github.com:armory-io/borealis-demo into argo-install

parth-srivastav committed:
create new borealis-argo namespace in setup script

parth-srivastav committed:
Merge pull request #10 from armory-io/argo-install

Argo install

armory-jenkins commented:
The title of this Pull Request needs to be changed to match the format:
```
<type>(<scope>): <subject>
```

See more details here: https://www.spinnaker.io/community/contributing/submitting/#commit-message-conventions

Some common examples are:
```
feat(armory.io): add docs.armory.io
fix(parser): grammar and spelling
style(loadingPage): made it super pretty!
refactor(helpMessages): changed order to make more sense
```

stephenatwell committed:
Merge branch 'setupdoogfood' into main

armory-jenkins commented:
The title of this Pull Request needs to be changed to match the format:
```
<type>(<scope>): <subject>
```

See more details here: https://www.spinnaker.io/community/contributing/submitting/#commit-message-conventions

Some common examples are:
```
feat(armory.io): add docs.armory.io
fix(parser): grammar and spelling
style(loadingPage): made it super pretty!
refactor(helpMessages): changed order to make more sense
```

Commit:
make setup runnabble

armory-jenkins commented:
The title of this Pull Request needs to be changed to match the format:
```
<type>(<scope>): <subject>
```

See more details here: https://www.spinnaker.io/community/contributing/submitting/#commit-message-conventions

Some common examples are:
```
feat(armory.io): add docs.armory.io
fix(parser): grammar and spelling
style(loadingPage): made it super pretty!
refactor(helpMessages): changed order to make more sense
```

Commit:
got cmdhook working for real, thus adding

stephenatwell committed:
Merge pull request #12 from armory-io/main

stephenatwell committed:
Merge pull request #13 from armory-io/argo-install

armory-jenkins commented:
The title of this Pull Request needs to be changed to match the format:
```
<type>(<scope>): <subject>
```

See more details here: https://www.spinnaker.io/community/contributing/submitting/#commit-message-conventions

Some common examples are:
```
feat(armory.io): add docs.armory.io
fix(parser): grammar and spelling
style(loadingPage): made it super pretty!
refactor(helpMessages): changed order to make more sense
```

parth-srivastav committed:
Added Lookback method to canary analysis.

stephenatwell committed:
Update setup.sh

stephenatwell committed:
Update setup.sh

Commit:
improved stability of setup script for provisioning empty clusters

bnewtonius committed:
fix(readme): setup must run in the config dir

bnewtonius committed:
Merge pull request #14 from armory-io/setup-script-fixes

fix(readme): setup must run in the config dir

Commit:
starting effort to get deploy.ymls to be kustomize generated so that we can more easily have multiple variants that reuse common bits

Commit:
added notifications to demo, and also updated the demo to generate the standard demo and the argo demo from common config via kustomize

Commit:
got kustomize working for argo as well

Commit:
now both work simultanesouly

stephenatwell committed:
Merge pull request #15 from armory-io/main

parth-srivastav committed:
Merge pull request #16 from armory-io/kustomize

feat (demo): added slack notifcations, made demo use kustomize to assemble deploy.ymls

stephenatwell committed:
Update webhookCallback.yml

stephenatwell committed:
Update checkLogs.yml

Commit:
Added previewservice to out standard demo configuration

Commit:
Merge branch 'kustomize' of https://github.com/armory-io/borealis-demo into kustomize

parth-srivastav committed:
Merge pull request #17 from armory-io/kustomize

feat(demo): Added previewservice to out standard demo configuration

Commit:
removed the loadbalancer services from the config, using preview service instead. Added preview service to the prod environments.

stephenatwell committed:
Merge pull request #19 from armory-io/main

Commit:
initial config for using clusterAPI controlplane to deploy CDaaS Demo envs

Commit:
Merge branch 'main' of https://github.com/armory-io/borealis-demo

Commit:
getting demo envs cloneable via clusterAPI

stephenatwell committed:
Merge pull request #21 from armory-io/main

stephenatwell committed:
Merge pull request #20 from armory-io/kustomize

stephenatwell committed:
Update targets.yml

christian noticed the service names in the traffic mgmt config no longer matched the deployed services

Commit:
demo setup via clusterAPI

Commit:
Merge remote-tracking branch 'refs/remotes/origin/main'
